en/tween/tween-interface.md

```diff
@@ -0,0 +1,80 @@
+# Tween Interface
+
+## Properties and Interface
+
+### Interface
+
+| Interface         | Description                                     |
+| :---------------- | :------------------------------------------ |
+| **tag**           | Add a tag of type `number` to the current tween |
+| **to**            | Adds an interval action that performs **absolute value** calculations on properties |
+| **by**            | Adds an interval action that performs **relative value** calculations on properties |
```

Comment:
Create motion by interpolate the property’s value to a value relative to the current one